src/Networking/v2/Models/LoadbalancerStat.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Networking\v2\Models;
+
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\OperatorResource;
+
+/**
+ * Represents Neutron v2 Loadbalancer Stats
+ *
+ * @property Api $api
+ */
+class LoadbalancerStat extends OperatorResource implements Retrievable
+{
+    /**
+     * @var integer
+     */
+    public $bytesIn;
+
+    /**
+     * @var integer
```

Comment:
Openstack uses [BigInt](https://github.com/openstack/neutron-lbaas/blob/9879460e23b84643720337dac7d9c47bedfe0aa1/tools/database-migration-from-v1-to-v2.py#L182) for `bytesIn` and `bytesOut`, unfortunately PHP does not have any equivalent data type.

I personally prefer using `string` in this case since the information here is read-only.

/cc @jamiehannaford